sonar-project.properties

```diff
@@ -0,0 +1,11 @@
+# The projectKey will need to be updated once we have this on CI
+sonar.projectKey=guolivei-rhub-api
+sonar.host.url=https://sonarqube.corp.redhat.com
```

Comment:
Did you successfully get a report for in sonarqube?

Reply:
Yes, I've created a key using my user to avoid noise on 'the real one'.
https://sonarqube.corp.redhat.com/dashboard?id=guolivei-rhub-api